providers/nitro/nitro-helper/src/config.rs

```diff
@@ -1,8 +1,8 @@
 use crate::shared::AwsCredentials;
+use clap::StructOpt;
```

Comment:
Shouldn't this be `clap::Parser`? Here's the reference documentation for `clap` derive macros: https://docs.rs/clap/latest/clap/_derive/index.html

Reply:
I followed https://github.com/clap-rs/clap/blob/v3-master/CHANGELOG.md#migrating
> 2. Replace your dependency from `structopt = "..."` to `clap = { version = "3.0", features = ["derive"] }`
> 3. Resolve compiler errors, including
> Update your `use` statements from `structopt` and `structopt::clap` to `clap`

And it seems to build fine?